sdks/python/setup.py

```diff
@@ -250,7 +250,7 @@ def get_portability_package_data():
         # Use a strict upper bound.
         'numpy>=1.14.3,<1.25.0',   # Update build-requirements.txt as well.
         'objsize>=0.6.1,<0.7.0',
-        'pymongo>=3.8.0,<4.0.0',
+        'pymongo>=3.8.0,<5.0.0',
```

Comment:
consider temporarily increasing the lower bound to 4.0.0 or above to make IT signal more reliable.